Celeste.Mod.mm/Content/Dialog/Polish.txt

```diff
@@ -18,232 +18,269 @@
-    OOBE_SETTINGS_OK=            OK
-
-# Mod Toggle Menu
-    MODOPTIONS_MODTOGGLE=                W(Y)ŁĄCZAJ MODY
-    MODOPTIONS_MODTOGGLE_LOADING=        Ładowanie informacji modów...
-    MODOPTIONS_MODTOGGLE_TOGGLEDEPS=    Przełączaj Zależności Automatycznie
-    MODOPTIONS_MODTOGGLE_TOGGLEDEPS_MESSAGE1=    Kiedy włączysz mod, wszystkie jego zależnosci zostaną włączone.
-    MODOPTIONS_MODTOGGLE_TOGGLEDEPS_MESSAGE2=    Kiedy wyłączysz mod, wszystkie mody które go wymagają zostaną wyłączone.
-    MODOPTIONS_MODTOGGLE_MESSAGE_1=        Jeśli włączysz lub wyłączysz mody, plik blacklist.txt zostanie zaktualizowany,
-    MODOPTIONS_MODTOGGLE_MESSAGE_2=        a Celeste zrestartowane by zastosować zmiany.
-    MODOPTIONS_MODTOGGLE_MESSAGE_3=        Podświetlone mody są używane przez inne mody jako zależności.
-    MODOPTIONS_MODTOGGLE_WHITELISTWARN= Wyłącz twoją białą listę aby te ustawienia zostały wprowadzone poprawnie.
-    MODOPTIONS_MODTOGGLE_ENABLEALL=        Włącz Wszystkie
-    MODOPTIONS_MODTOGGLE_DISABLEALL=    Wyłącz Wszystkie
-    MODOPTIONS_MODTOGGLE_CANCEL=        Anuluj
-    MODOPTIONS_MODTOGGLE_ZIPS=            Pliki Zip
-    MODOPTIONS_MODTOGGLE_DIRECTORIES=    Foldery
-    MODOPTIONS_MODTOGGLE_BINS=            Pliki Bin Map
-
-# Asset Reload Helper
-    ASSETRELOADHELPER_RELOADINGMAP=            Odświeżanie mapy
-    ASSETRELOADHELPER_RELOADINGLEVEL=        Odświeżanie poziomu
-    ASSETRELOADHELPER_UNLOADINGTEXTURE=        Wyłączanie tekstury:
-    ASSETRELOADHELPER_RELOADINGMAPNAME=        Odświeżanie mapy:
-    ASSETRELOADHELPER_RELOADINGALLMAPS=        Odświeżanie wszystkich map
-    ASSETRELOADHELPER_RELOADINGDIALOG=        Odświeżanie dialogu:
-    ASSETRELOADHELPER_RELOADINGTEXTURE=        Odświeżanie tekstury:
-    ASSETRELOADHELPER_RELOADINGTEXTURES=    Odświeżanie tekstur:
-    ASSETRELOADHELPER_LOADINGNEWMOD=        Ładowanie nowego moda:
-    ASSETRELOADHELPER_RELOADINGMODASSEMBLY=    Odświeżanie asemblacji modów:
-    ASSETRELOADHELPER_RELOADINGMOUNTAIN=    Odświeżanie góry
+# Helper Postcards // Pocztówki Wspomagające
+	POSTCARD_LEVELGONE=			{big}Dziękuję ((player))!{/big}{n}Ale niestety {#ff1144}((sid)){#} jest na innej górze.
+	POSTCARD_LEVELLOADFAILED=	{big}Przykro mi!{/big}{n}Ale nie udało się załadować {#ff1144}((sid)){#}.{n}Wyślij swój {#44adf7}log.txt{#}{n}do twórcy mapy.
+	POSTCARD_LEVELNOSPAWN=		{big}Ups!{/big}{n}Ktoś zagubił {#44adf7}punkt startowy!{#}{n}Sprawdź, czy twoja mapa{n}zawiera obiekt typu {#44adf7}Player{#}.
+	POSTCARD_BADTILEID=			{big}Ups!{/big}{n}Znaleziono niezdefiniowany kafelek ((type)) o id {#ff1144}((id)){#} w punkcie{n}{#ff1144}(((x)), ((y))){#} w pokoju {#ff1144}((room)){#}.{n}Sprawdź swój {#44adf7}log.txt{#} by uzyskać więcej informacji.
+	POSTCARD_DECALREGERROR=		{big}Ups!{/big}{n}Wystąpił błąd podczas zastosowywania{n}własności {#ff1144}((property)){#} dla {#ff1144}((decal)){#}{n}Być może masz brakującą lub nieprawidłową własność w swoim {#44adf7}Rejestrze Kalkomanii{#}.
```

Comment:
> Rejestr Kalkomanii

Huh? It would probably be best to just leave it as Decal Registry imo, this name makes no sense at all to me, and the file has to be named DecalRegistry.xml anyway

Reply:
Resolved in #modding_dev.